src/test/java/seedu/giatros/logic/parser/FindCommandParserTest.java

```diff
@@ -6,15 +6,24 @@
 
 import java.util.Arrays;
 
+import org.junit.BeforeClass;
 import org.junit.Test;
 
+import seedu.giatros.commons.core.EventsCenter;
+import seedu.giatros.commons.events.ui.accounts.LoginEvent;
 import seedu.giatros.logic.commands.FindCommand;
 import seedu.giatros.model.patient.NameContainsKeywordsPredicate;
+import seedu.giatros.ui.testutil.AccountCreator;
 
 public class FindCommandParserTest {
 
     private FindCommandParser parser = new FindCommandParser();
 
+    @BeforeClass
```

Comment:
To ensure that test is run under manager

Reply:
Yes